Makefile

```diff
@@ -0,0 +1,35 @@
+CDH_COMMIT ?=
+
+default_cdh_commit := ed809e5
+
+ifeq ($(CDH_COMMIT),)
+    cdh_commit := $(default_cdh_commit)
+else
+    cdh_commit := $(CDH_COMMIT)
+endif
+
+no_cache :=
+ifneq ($(findstring $(cdh_commit),$(default_cdh_commit)),$(default_cdh_commit))
+    no_cache := --no-cache
+endif
+
+.PHONE: all build build-image-grpc build-image-ttrpc
+
+all: help
+
+help: ## Print this help information
+	@awk -F ':|##' '/^[^\t].+?:.*?##/ { printf "\033[36m%-20s\033[0m", $$1; for (i=3; i<=NF; i++) printf " %s", $$i; printf "\n" }' $(MAKEFILE_LIST)
+
+build: build-grpc build-ttrpc ## Build the confidential-data-hub images with CDH_COMMIT to set the commit to build
```

Comment:
could we move the comments one line before this?

Reply:
Do you mean that moving the comments to a seperate line of the code?
I tried that but after moving the comments to a seperate line of the code, by typing `make help` there will be no description of each command, which means keeping comments at the same line of `help`, `build`... is helpful for ease of usage.

Reply:
Ok. sounds reasonable